Firmware/Marlin_main.cpp

```diff
@@ -2583,7 +2583,8 @@ static void gcode_G80()
 {
     constexpr float XY_AXIS_FEEDRATE = (homing_feedrate[X_AXIS] * 3) / 60;
     constexpr float Z_LIFT_FEEDRATE = homing_feedrate[Z_AXIS] / 60;
-    constexpr float Z_CALIBRATION_THRESHOLD = 0.35f;
+    constexpr float Z_CALIBRATION_THRESHOLD_TIGHT = 0.35f; // used for 7x7 MBL
+    constexpr float Z_CALIBRATION_THRESHOLD_RELAXED = 1.f; // used for 3x3 MBL
```

Comment:
Would it make sense to move these into `Configuration_adv.h`? I'm OK with keeping these here, just a thought.

Reply:
Hmmm I guess we could. Even before my changes in 3.14.0 it wasn't extracted, but I don't oppose the change. I'll move it next to the other MBL parameters later today

Reply:
Nevermind. There are no MBL related configs in Configuration.h and Configuration_adv.h. I really don't want to add this to the variants config. I don't feel like users are expected to have to change these settings